### ISSUE TITLE
Leave some jobs for others

### ISSUE THREAD

**Issue by Joshua260403** [richelbilderbeek/djog_unos_2018#110]
I noticed alot of jobs are assigned to the same person, leaving just a few jobs let for all the others.

Maybe we can set a max of 1 job self assigned at a time.
An exception can be made for jobs nobody wants, but even then everyone needs to get a chance to do something.

**Comment by richelbilderbeek** [comment]
Indeed, [there are people that do more than others](https://github.com/richelbilderbeek/djog_unos_2018/graphs/contributors), with @RafayelGardishyan in the lead. 

I can only welcome this, as we have little time. AFAICS, the main devs (you and Rafayel) do not reserve all jobs for themselves, but just do many jobs after another. I may be wrong, as I do not monitor the 'In Progress' column throughout a day that I must do work myself.

Still, I see the reason you bring up this point. It would result in an imbalanced team.

What do you suggest me to do? Your suggested rule is AFAICS mostly respected and me enforcing it a bit more would not matter much. Letting the main devs teach more would help, but not on a Saturday.

Thanks for bringing up this point and I will think about this. Looking forward to your ideas. Also, @RafayelGardishyan, what do you think of the point brought up? Do you see a solution?

**Comment by richelbilderbeek** [comment]
@Joshua260403 and @RafayelGardishyan: I have been thinking about this Issue. I had the following insight:

```
medior devs should allow junior devs to become junior devs
```

and likewise:

```
senior devs should allow medior devs to become senior devs
```

As I observe there is at least one job I should assign to the mediors.
 
I suggest to 

 * label all Issues with `junior` and `medior`. 
 * Sometimes I will add to the Issue a line like `Not to be done by [name]`

Great plan?

**Comment by richelbilderbeek** [comment]
I observed that @Joshua260403 had a point, that multiple people (including him :grin:) assigned themselves multiple Issues. I ruthlessly undid that. I suggest to follow the rule:

> As a rule, you will have one Issue assigned
> As an exception, Richel will assign additional Issues

I will close this Issue, as @Joshua260403 correctly spotted I did not observe multiple users assigned themselves multiple Issues. My bad! I will pay more attention this this from now on.
